CONTRIBUTING.md

```diff
@@ -28,7 +28,7 @@ restrictions:
   [`Normalize`](https://github.com/necolas/normalize.css) (open them in
   their respective repositories).
 
-* Please **do not** open issues regarding the official themes offered on <http://themes.getbootstrap.com/>. Instead, please email any questions or feedback regarding those themes to `themes AT getbootstrap DOT com`.
+* Please **do not** open issues regarding the official themes offered on <http://themes.getbootstrap.com/>. Instead, please email any questions or feedback regarding those themes to <themes@getbootstrap.com>.
```

Comment:
Oh interesting... To prevent from spammers that scrape GitHub for email addresses?
I'll revert this change.